FileStreamRotator.js

```diff
@@ -480,10 +480,12 @@ FileStreamRotator.getStream = function (options) {
         var f = null;
         if(auditLog && auditLog.files && auditLog.files instanceof Array && auditLog.files.length > 0){
             var lastEntry = auditLog.files[auditLog.files.length - 1].name;
-            if(lastEntry.match(t_log)){
-                var lastCount = lastEntry.match(t_log + "\\.(\\d+)");
+            var lastEntryWithoutPath = lastEntry.split("\\").pop();
+            var t_logWithoutPath = t_log.split("\\").pop();
```

Comment:
@yakirzana I wonder if you want to substitute `"\\"` with `path.sep` to return the platform-specific file separator.

Reply:
Yes, sure.